Add a note that "Copy link" is not a share for DataPackage (#2331)
MicrosoftDocs/winrt-api: windows.applicationmodel.datatransfer/datapackage_sharecompleted.md

An event that is triggered when a share is completed. Shares can be sent to an app, a provider, or a contact.

## -remarks

> Note that selecting "Copy link" does not count as a share and thus does not fire this event, nor ShareCanceled.

Reviewer: ```suggestion
> [!NOTE]
> Selecting "Copy link" is not considered a *share* and thus does not fire this event, nor `ShareCanceled`.
```
Added some formatting to have the note displayed as a "note" in markdown. I also tweaked the wording.
